config/crd/bases/control.k8ssandra.io_cassandratasks.yaml

```diff
@@ -1,9 +1,10 @@
+
 ---
 apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.10.0
+    controller-gen.kubebuilder.io/version: v0.6.1
```

Comment:
You need to regenerate all these, as they're using the wrong controller-gen version.